db/migrate/20230919141119_create_projects.rb

```diff
@@ -1,7 +1,7 @@
 class CreateProjects < ActiveRecord::Migration[6.1]
   def change
     create_table :projects do |t|
-      t.string :name, null: false
+      t.string :name, null: false, unique: true
```

Comment:
This is bad solution, this migration already in master, you need to create new migrations